src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+* {
+    margin: 0;
+
+}

```

Comment:
it's not a good practice to use `*` selector for resetting margin. just reset for `body` tag
```suggestion
```